internal/policygen/schema.go

```diff
@@ -16,7 +16,7 @@ package policygen
 
 // TODO(https://github.com/golang/go/issues/35950): Move this to its own file.
 
-const schema = `
+var schema = []byte(`
```

Comment:
Does this need to be changed? You're not modifying this var it seems

Reply:
It seems I can't make a byte slice an const

Reply:
and then the byte array is directly used in the hcl.ToJSON func